frontend/packages/git-service/src/utils/import-strategy-detector.ts

```diff
@@ -87,8 +104,14 @@ export const detectImportStrategies = async (
     loaded = true;
   }
 
+  if (!isPacRepository(isRepositoryEnabled, gitService, pacFiles)) {
+    detectedStrategies = detectedStrategies.filter(
+      (strategy) => strategy.type !== ImportStrategy.PAC,
+    );
+  }
+
   detectedStrategies = detectedStrategies
-    .filter((strategy) => !!strategy.detectedFiles.length || !!strategy.detectedCustomData?.length)
+    .filter((strategy) => !!strategy.detectedFiles.length || !!strategy?.detectedCustomData?.length)
```

Comment:
If `strategy` is undefined on the right side it would already crash on the left side, so this change looks unnecessary to me:

```suggestion
    .filter((strategy) => !!strategy.detectedFiles.length || !!strategy.detectedCustomData?.length)
```

Reply:
Okay. Reverting it